test/gpdb/test_in_docker.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
do we want to see the progress of this script in the logs? (`-x`)

Reply:
I wanted to have pretty echo output...